src/theme/ExternalLinkIcon/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import styles from "./styles.module.css";
+
+function IconExternalLink({ width = 15, height = 15 }) {
```

Comment:
Oops. I named the component one thing (Icon...) and the file something else (External...). I'll push up a fix to synchronize this.